spec/support/feature_steps.rb

```diff
@@ -209,6 +204,8 @@ def fill_claim
 
   def fill_et_claim
     fill_in 'claim_et_identifier', with: '12345'
+    # for some reason this has to be clicked twice.
+    click_button 'Continue'
```

Comment:
@colinbruce I know I was trying to find out if it's js related or something but it doesn't look like it. I will have a deeper look later on.